components/search/SearchInterface.vue

```diff
@@ -295,7 +302,34 @@
           }
         }
         return updated;
+      },
+      clearFilters() {
+        let qf = this.$route.query.qf;
+
+        const result = qf.filter(item => {
+          const key = item.split(':')[0];
+
+          return !defaultFacets.includes(key);
+        });
+
+        this.rerouteSearch({ qf: result, reusability: null });
       }
     }
   };
 </script>
+
+<style lang="scss" scoped>
+  @import "./assets/scss/variables.scss";
+  @import "./assets/scss/icons.scss";
+
+  .clear-all {
+    background: none;
+    border: none;
+    color: $lightblue;
```

Comment:
this causes an accessibility violation due to colour contrast. Changing it to $blue resolves it and is still more or less in line with the design.